funnel/forms/venue.py

```diff
@@ -1,12 +1,12 @@
 # -*- coding: utf-8 -*-
 
+import pycountry
 from baseframe import __
 import baseframe.forms as forms
 from baseframe.forms.sqlalchemy import QuerySelectField
-from baseframe.staticdata import country_codes
 from .project import valid_color_re
-from ..models import Venue
 
+country_codes = [(country.alpha_2, country.name) for country in pycountry.countries]
```

Comment:
This should be below `__all__`

Reply:
Also, can the names be localized? This will be a roadblock when we introduce a second language in Funnel.

Reply:
Lastly, this should be sorted by country name. Is that a guarantee from pycountry? If not, we should be doing it here.

Reply:
It's already sorted by pycountry. And pycountry has locales built-in, but not sure how to integrate that.